flybywiresim/a32nx
# fix(mcdu): remove replacement chars on aoc atis page

Fixes #[issue_no]

## Summary of Changes

Small fix that removes the unicode replacement characters from AOC ATIS REQUEST page.

## Screenshots (if necessary)


**Before:**
![Microsoft Flight Simulator Screenshot 2021 04 25 - 00 04 59 68](https://user-images.githubusercontent.com/58574351/115974306-f7a00b00-a55b-11eb-82de-1e118507a763.png) ![Microsoft Flight Simulator Screenshot 2021 04 25 - 00 05 07 30](https://user-images.githubusercontent.com/58574351/115974312-fbcc2880-a55b-11eb-97d7-5544280af2f2.png) ![Microsoft Flight Simulator Screenshot 2021 04 25 - 00 05 15 04](https://user-images.githubusercontent.com/58574351/115974316-0090dc80-a55c-11eb-9434-e84cb1dd9029.png)

**After:**
![Microsoft Flight Simulator Screenshot 2021 04 25 - 00 06 31 31](https://user-images.githubusercontent.com/58574351/115974325-130b1600-a55c-11eb-8029-fd8cb3b6123d.png) ![Microsoft Flight Simulator Screenshot 2021 04 25 - 00 06 38 76](https://user-images.githubusercontent.com/58574351/115974327-16060680-a55c-11eb-9a9b-53cb002061b3.png) 
![Microsoft Flight Simulator Screenshot 2021 04 25 - 00 06 47 57](https://user-images.githubusercontent.com/58574351/115974334-1bfbe780-a55c-11eb-9d6b-ea7898453320.png)


Discord username (if different from GitHub): `ExampleWasTaken#0886`

## Testing instructions

- Make sure that the replacement characters are not displayed.
- Make sure the ATIS request feature still works. (It is important that you request the ATIS from an airport that currently has an ATIS! Check [this](https://vau.aero/fsmap/?osm) site for available ATISs)


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
